Formula/rust.rb

```diff
@@ -88,28 +87,23 @@ def install
 
     if build.with? "racer"
       resource("racer").stage do
-        (buildpath/"cargo_home").mkdir
-        ENV["PATH"] = ENV["PATH"]+":"+bin
-        ENV["CARGO_HOME"] = buildpath/"cargo_home"
-        # interactive_shell
-        system (bin/"cargo"), "build", "--release", "--verbose"
+        ENV.append_path "PATH", bin
+        cargo_home = buildpath/"cargo_home"
+        cargo_home.mkpath
+        ENV["CARGO_HOME"] = cargo_home
+        system bin/"cargo", "build", "--release", "--verbose"
         (libexec/"bin").install "target/release/racer"
-        (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => pkgshare/"rust_src/current")
+        (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => pkgshare/"rust_src")
       end
+      # remove any binary files as they break homebrew, keep everything else
```

Comment:
They need not be libraries to cause the problem. It's any binary file at all that goes through the "install" procedure.